.buildkite/scripts/dra_release.sh

```diff
@@ -11,6 +11,7 @@ export PROJECT="fleet-server"
 export TYPE=${1}
 export BRANCH="${BUILDKITE_BRANCH}"
 export VERSION="$(make get-version)"
+export ${WORKSPACE}
```

Comment:
Should not be just `export WORKSPACE`

Reply:
I checked it with $, and NP, changed it, please check it

Reply:
just tried to use one approach ))

Reply:
just tested with this:
```
 $ WORKSPACE="testtest"
 $ export |grep WORKSPACE
 $ 
 $ export ${WORKSPACE}
 $ export |grep WORKS
 $ export |grep testtest
declare -x testtest
 $ export WORKSPACE
 $ export |grep WORKS
declare -x WORKSPACE="a"
```

if it is used `${}` , it's being exported the contents of the env. var